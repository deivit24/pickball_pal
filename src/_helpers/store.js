import { createStore, applyMiddleware } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import rootReducer from '../_reducers/root';

//Creating the store
const store = createStore(
  rootReducer,
  composeWithDevTools(applyMiddleware(thunk))
);

export default store;
