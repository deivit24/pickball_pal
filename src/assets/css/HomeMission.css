#HomeMission {
  background-color: #e4e4e4;
  padding: 200px 0px;
}
#HomeMission img {
  width: 100%;
}
#HomeMission .col-md-6:last-child {
  text-align: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
  padding: 0 30px;
}
#HomeMission .col-md-6 h1 {
  font-weight: 700;
}
