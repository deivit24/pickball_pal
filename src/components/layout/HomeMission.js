import React from 'react';
import img from '../../assets/img/pickleball.jpg';

import '../../assets/css/HomeMission.css';

const HomeMission = () => {
  return (
    <div id="HomeMission">
      <div className="container">
        <div className="row">
          <div className="col-md-6">
            <img src={img} alt="" />
          </div>
          <div className="col-md-6">
            <h1>We're on a mission</h1>
            <p>
              Our goal is to grow Pickleball by making it easier to connect with
              players, get practice, and play matches at your local courts
            </p>
          </div>
        </div>
      </div>
    </div>
  );
};

export default HomeMission;
