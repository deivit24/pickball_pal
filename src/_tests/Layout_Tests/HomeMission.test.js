import React from 'react';
import { render } from '@testing-library/react';
import HomeMission from '../../components/layout/HomeMission';
import { MemoryRouter } from 'react-router';
import { Provider } from 'react-redux';
import store from '../../_helpers/store';

//Testing HomeMission

describe('Testing the HomeMission', () => {
  it('it renders HomeMission without crashing', () => {
    render(
      <MemoryRouter>
        <Provider store={store}>
          <HomeMission />
        </Provider>
      </MemoryRouter>
    );
  });

  it('matches snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Provider store={store}>
          <HomeMission />
        </Provider>
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
