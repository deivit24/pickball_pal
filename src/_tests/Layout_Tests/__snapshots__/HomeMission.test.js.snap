// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the HomeMission matches snapshot 1`] = `
<DocumentFragment>
  <div
    id="HomeMission"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md-6"
        >
          <img
            alt=""
            src="pickleball.jpg"
          />
        </div>
        <div
          class="col-md-6"
        >
          <h1>
            We're on a mission
          </h1>
          <p>
            Our goal is to grow Pickleball by making it easier to connect with players, get practice, and play matches at your local courts
          </p>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
